selection-scale-rotate.user.js

```diff
@@ -1,13 +1,13 @@
 // ==UserScript==
 // @name         Line Rider Selection Rotate and Scale Mod
 // @namespace    http://tampermonkey.net/
-// @version      0.4
+// @version      0.5
 // @description  Adds ability to rotate and scale selections
-// @author       David Lu
+// @author       David Lu & Ethan Li
 // @match        https://www.linerider.com/*
 // @match        https://*.official-linerider.com/*
 // @match        http://localhost:8000/*
-// @downloadURL  https://github.com/EmergentStudios/linerider-userscript-mods/raw/master/selection-scale-rotate.user.js
+// @downloadURL  https://github.com/ethanjli/linerider-userscript-mods/raw/master/selection-scale-rotate.user.js
```

Comment:
Given that this will exist in `EmergentStudios/linerider-userscript-mods`, the download URL should continue to point here, right?

Reply:
Yes, when I committed 71a3b35 I wasn't sure if this repo was still going to be maintained. I'll change the download URL back when I also merge https://github.com/ethanjli/linerider-userscript-mods/pull/2 into this PR.

Reply:
Ok, I've reverted the download URL and merged https://github.com/ethanjli/linerider-userscript-mods/pull/2 into this PR - please let me know if there's anything else I should do on this PR!